btk/create_observing_generator.py

```diff
@@ -7,6 +7,7 @@ def __init__(
         survey_name,
         obs_conds=None,
         verbose=False,
+        multiresolution=False
```

Comment:
In my original design I had wondered whether a flag would be useful and finally kept it but in retrospect I think it can be removed and replaced by a check on the nature of survey_name

I heard about some Rubin-Euclid-Roman joint analysis so you never know.

Reply:
haha ok, we can keep the `len` unconstrained then